tests/capture/test_templates.py

```diff
@@ -686,6 +688,66 @@ def qfunc():
         assert len(q) == 1
         qml.assert_equal(q.queue[0], qml.QROM(**kwargs))
 
+    @pytest.mark.usefixtures("new_opmath_only")
+    def test_phase_adder(self):
+        """Test the primitive bind call of PhaseAdder."""
+
+        k = 3
+        wires = [0, 1, 2]
+
+        def qfunc(k, wires):
+            qml.PhaseAdder(k, wires)
+
+        # Validate inputs
+        qfunc(k, wires)
+
+        # Actually test primitive bind
+        jaxpr = jax.make_jaxpr(qfunc)(k, wires)
+
+        assert len(jaxpr.eqns) == 1
+
+        eqn = jaxpr.eqns[0]
+        assert eqn.primitive == qml.PhaseAdder._primitive
+        assert eqn.invars == jaxpr.jaxpr.invars
+        assert len(eqn.outvars) == 1
+        assert isinstance(eqn.outvars[0], jax.core.DropVar)
+
+        with qml.queuing.AnnotatedQueue() as q:
+            jax.core.eval_jaxpr(jaxpr.jaxpr, jaxpr.consts)
+
+        assert len(q) == 1
+        assert qml.equal(q.queue[0], qml.PhaseAdder(k, wires))
+
+    @pytest.mark.usefixtures("new_opmath_only")
+    def test_adder(self):
+        """Test the primitive bind call of Adder."""
+
+        k = 3
+        wires = [0, 1, 2]
+
+        def qfunc(k, wires):
+            qml.Adder(k, wires)
+
+        # Validate inputs
+        qfunc(k, wires)
+
+        # Actually test primitive bind
+        jaxpr = jax.make_jaxpr(qfunc)(k, wires)
+
+        assert len(jaxpr.eqns) == 1
+
+        eqn = jaxpr.eqns[0]
+        assert eqn.primitive == qml.Adder._primitive
+        assert eqn.invars == jaxpr.jaxpr.invars
+        assert len(eqn.outvars) == 1
+        assert isinstance(eqn.outvars[0], jax.core.DropVar)
+
+        with qml.queuing.AnnotatedQueue() as q:
+            jax.core.eval_jaxpr(jaxpr.jaxpr, jaxpr.consts)
+
+        assert len(q) == 1
+        assert qml.equal(q.queue[0], qml.Adder(k, wires))
+
```

Comment:
No need for dedicated tests, instead add test cases to `unmodified_templates_cases`
```suggestion
```

Reply:
Why these tests are not removed?